src/components/earn-opportunities-main-info/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { BigNumber } from 'bignumber.js';
+import { noop } from 'lodash-es';
+import React, { FC } from 'react';
+import { View, Text } from 'react-native';
+
+import { Button } from 'src/components/button/button';
+import { Divider } from 'src/components/divider/divider';
+import { FormattedAmount } from 'src/components/formatted-amount';
+import { DEFAULT_AMOUNT, DEFAULT_DECIMALS, PENNY } from 'src/config/earn-opportunities-main-info';
+import { EmptyFn } from 'src/config/general';
+import { useCurrentFiatCurrencyMetadataSelector } from 'src/store/settings/settings-selectors';
+import { formatSize } from 'src/styles/format-size';
+
+import { useEarnOpportunitiesMainInfoStyles, useButtonPrimaryStyleConfig } from './styles';
+
+interface Props {
+  claimAllRewards?: EmptyFn;
+  shouldShowClaimRewardsButton?: boolean;
+  totalClaimableRewardsInFiat?: BigNumber;
+  netApr: BigNumber;
+  totalStakedAmountInFiat: BigNumber;
+  areSomeRewardsClaimable?: boolean;
+}
+
+export const EarnOpportunitiesMainInfo: FC<Props> = ({
+  claimAllRewards = noop,
+  shouldShowClaimRewardsButton = false,
+  totalClaimableRewardsInFiat = DEFAULT_AMOUNT,
+  netApr,
+  totalStakedAmountInFiat,
+  areSomeRewardsClaimable = false
+}) => {
+  const styles = useEarnOpportunitiesMainInfoStyles();
+  const buttonPrimaryStylesConfig = useButtonPrimaryStyleConfig();
+  const { symbol: fiatSymbol } = useCurrentFiatCurrencyMetadataSelector();
+  const roundingMode = totalClaimableRewardsInFiat?.isLessThan(PENNY) ? BigNumber.ROUND_UP : undefined;
+
+  return (
+    <View style={styles.root}>
+      <View>
+        <View style={styles.container}>
+          <View style={[styles.card, styles.deposit]}>
+            <Text style={styles.titleText}>CURRENT DEPOSIT AMOUNT</Text>
+            <FormattedAmount isDollarValue amount={totalStakedAmountInFiat} style={styles.valueText} />
+          </View>
+          <Divider size={formatSize(8)} />
+          <View style={[styles.card, styles.netApr]}>
+            <Text style={styles.titleText}>NET APR</Text>
+            <Text style={styles.valueText}>{netApr.toFixed(DEFAULT_DECIMALS)}%</Text>
+          </View>
+        </View>
+        {shouldShowClaimRewardsButton && (
+          <>
+            <Divider size={formatSize(16)} />
+            <Button
+              styleConfig={buttonPrimaryStylesConfig}
+              title={
+                areSomeRewardsClaimable
```

Comment:
I had a misunderstanding with naming. 
if _some_ rewards are claimable why the title is "claim All" ?
if we can claim only all rewards, maybe better name the variable as `areRewardsClaimable` ?

Reply:
Actually, a 'Claim all' button starts claiming rewards only from those farms/savings where the vesting period is over. A card for a Quipuswap farm has a 'Claim rewards' button, which starts claiming rewards from the specified farm disregarding its vesting period.

Reply:
However, `areRewardsClaimable` looks like a better option for the name